### PR TITLE
Added idNumber for nl_NL

### PR DESCRIPTION
**Pull request by artorozenga**
- Added personal identification number

### CODE REVIEW

File: src/Faker/Provider/nl_NL/Person.php [hunk 1]
```diff
@@ -314,4 +314,33 @@ public static function prefix()
     {
         return static::randomElement(static::$prefix);
     }
+    
+    /**
+     * 11 proof idNumber
```
Comment (fzaninotto) [inline src/Faker/Provider/nl_NL/Person.php]: please add a `@link` annotation to a Wikipedia-like description of this id number, and of the algo you used here.

File: src/Faker/Provider/nl_NL/Person.php [hunk 2]
```diff
@@ -314,4 +314,33 @@ public static function prefix()
...
+     * @return string
+     */
+    public function idNumber()
+    {
+        $return = '';
+        $nr = [];
+        $nr[] = 0;
+        while(count($nr) < 8) {
+            $nr[] = rand(0,9);
```
Comment (fzaninotto) [inline src/Faker/Provider/nl_NL/Person.php]: never use `rand()` in Faker, use `mt_rand()` instead. And if you want to generate a random digit, use `randomDigit` instead. Same on line 330

File: src/Faker/Provider/nl_NL/Person.php [hunk 3]
```diff
@@ -314,4 +314,33 @@ public static function prefix()
...
+            if( $nr[1]>0) {
+                $nr[0]=8;
+                $nr[1]--;
+            } else {
+                $nr[0]=1;
+                $nr[1]++;
+
+            }
+        }
+
```
Comment (fzaninotto) [inline src/Faker/Provider/nl_NL/Person.php]: missing return ?

**Comment by fzaninotto** [comment]
Still missing a mention in the locale-specific section of the README

**Comment by artorozenga** [comment]
Updated the readme

File: readme.md [hunk 4]
```diff
@@ -1276,6 +1276,14 @@ echo $faker->vat; // "NL123456789B01" - Dutch Value Added Tax number
 echo $faker->btw; // "NL123456789B01" - Dutch Value Added Tax number (alias)
 ```
 
+### `Faker\Provider\nl_NL\Person`
+
+```php
+<?php
+
+echo $faker->idNumber; // "111222333" - Dutch Personal identificatin number (BSN)
```
Comment (fzaninotto) [inline readme.md]: typo identificatin => identification

**Comment by fzaninotto** [comment]
Thanks!

**Pull request merged**